packages/core/src/internal/data-grid/data-grid.tsx

```diff
@@ -1034,6 +1038,11 @@ const DataGrid: React.ForwardRefRenderFunction<DataGridRef, DataGridProps> = (p,
                     ...args,
                     isLongTouch: true,
                 };
+            } else if (!args.isTouch && lastUpTimeValue !== 0 && Date.now() - lastUpTimeValue < 500) {
```

Comment:
If `double-click` is configured, cells cannot be activated on a touch device or?

Reply:
Good catch that is screwed up.

Reply:
Fixed